openupgrade_framework/odoo_patch/odoo/addons/base/models/ir_ui_view.py

```diff
@@ -34,16 +34,16 @@ def _raise_view_error(
                 from_exception=from_exception,
                 from_traceback=from_traceback,
             )
-        except ValueError:
+        except ValueError as e:
             _logger.warning(
                 "Can't render custom view %s for model %s. "
-                "Assuming you are migrating between major versions of "
-                "Odoo, this view is now set to inactive. Please "
-                "review the view contents manually after the migration.",
+                "Assuming you are migrating between major versions of Odoo. "
+                "Please review the view contents manually after the migration.\n"
+                "Error: %s",
                 self.xml_id,
                 self.model,
+                e,
```

Comment:
One last doubt. Why including `e` in the log? Isn't this too verbose?

Reply:
So that we can review faster, otherwise you won't know why it failed

Reply:
This is not answering my question. Isn't this too verbose? Can you put an example of the output with the exception included in the log message?